src/Symfony/Installer/NewCommand.php

```diff
@@ -137,7 +137,7 @@ private function checkSymfonyVersionIsInstallable()
             return $this;
         }
 
-        // validate server syntax
+        // validate semver syntax
```

Comment:
I would actually change this to "validate version syntax" or "validate version pattern".